src/group_theory/quotient_group.lean

```diff
@@ -229,16 +229,30 @@ end
 noncomputable def quotient_ker_equiv_range : (quotient (ker φ)) ≃* range φ :=
 mul_equiv.of_bijective (range_ker_lift φ) ⟨range_ker_lift_injective φ, range_ker_lift_surjective φ⟩
 
-/-- The canonical isomorphism `G/(ker φ) ≃* H` induced by a surjection `φ : G →* H`. -/
+/-- The canonical isomorphism `G/(ker φ) ≃* H` induced by a homomorphism `φ : G →* H`
+with a right inverse `ψ : H →* G`. -/
```

Comment:
```suggestion
with a right inverse `ψ : H → G`. -/
```
Unless you're making a mathematical comment not reflected by the lean. Same for the other 3 docstrings.

Reply:
(You can prove that `ψ` is in fact a `monoid_hom`, but I agree that we should stay consistent with the code.)